### PR TITLE
PowerBI Dashboard Doc update

### PR DESCRIPTION
**Pull request by v-sambati**
Uploaded .zip for .pbix
updated steps for replacing data source

### CODE REVIEW

**Comment by PRMerger7** [comment]
@v-sambati : Thanks for your contribution! The author(s) have been notified to review your proposed change. @nikotha

**Comment by PRMerger13** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Image File | This PR contains an image file(s).

File: microsoft-community-training/analytics/custom-reports/power-bi-integration.md [hunk 1]
```diff
@@ -57,10 +59,49 @@ In this article, you will learn more about connecting Power BI with the Azure SQ
...
 6. Once you hit on load all your selected tables will be imported into Power Bi right pane and available under fields.
+
     :::image type="content" source="../../media/image%28367%29.png" alt-text="Loaded tables will be imported to PowerBI":::
 
 7. Use the database schema from [this document](database-schema.md) to build relations required for your custom reporting requirements
 
->[!Tip]
->To get started, MCT provides customers with custom visualizations through Power BI Dashboard to effectively track learner’s progress across courses.
->[**MCT PowerBI custom Dashboard.zip**](https://github.com/MicrosoftDocs/microsoft-community-training/files/6967550/MCT.PowerBI.custom.Dashboard.zip) this folder contains custom PowerBI dashboard and a file with steps to replace the existing data source with your instance's database to view out of the box insights from your data through custom created visualizations.
+## Using custom Power BI dashboard
```
Comment (nikotha) [inline microsoft-community-training/analytics/custom-reports/power-bi-integration.md:67]: The header can be made as "Sample PowerBI Reports"

File: microsoft-community-training/analytics/custom-reports/power-bi-integration.md [hunk 2]
```diff
@@ -57,10 +59,49 @@ In this article, you will learn more about connecting Power BI with the Azure SQ
...
+
+To get started, MCT provides customers with a custom made PowerBI dashboard with interactive visualizations to effectively track learner’s progress across courses.
```
Comment (nikotha) [inline microsoft-community-training/analytics/custom-reports/power-bi-integration.md:69]: To help you get started on creating your own PowerBI reports here are some samples. These sample reports with their interactive visualizations can be used to create further customized reports for your specific requirements. 

The pbix files for the sample reports can be downloaded from here -

File: microsoft-community-training/analytics/custom-reports/power-bi-integration.md [hunk 3]
```diff
@@ -57,10 +59,49 @@ In this article, you will learn more about connecting Power BI with the Azure SQ
...
+
+>[**MCT PowerBI custom Dashboard.zip**](https://github.com/MicrosoftDocs/microsoft-community-training/files/6967550/MCT.PowerBI.custom.Dashboard.zip)
+
+To be able to start using this dashboard we need to replace the existing data source and connect the .Pbix file with your own [**Database**](database-schema.md).
```
Comment (nikotha) [inline microsoft-community-training/analytics/custom-reports/power-bi-integration.md:73]: replace "we need to" with "you will need to"

Comment (nikotha) [reply microsoft-community-training/analytics/custom-reports/power-bi-integration.md:73]: connect the above downloaded .pbix file

**Review by nikotha — CHANGES_REQUESTED** [review]

**Comment by PRMerger7** [comment]
@v-sambati : Thanks for your contribution! The author(s) have been notified to review your proposed change. @nikotha

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Image File | This PR contains an image file(s).

**Pull request merged**